cargo-dist/src/lib.rs

```diff
@@ -372,7 +373,17 @@ fn build_cargo_target(dist_graph: &DistGraph, target: &CargoBuildStep) -> Result
         target.target_triple, target.profile
     );
 
-    let mut command = Command::new(&dist_graph.tools.cargo.cmd);
+    let cargo = &dist_graph.tools.cargo.cmd;
+    let skip_brewfile = env::var("DO_NOT_USE_BREWFILE").is_ok();
+
+    let mut command;
+    if Utf8Path::new("Brewfile").exists() && !skip_brewfile {
+        command = Command::new("brew");
```

Comment:
Can we register `brew` as an optional `dist_graph.tools` item (like `rustup`), and also make this path conditional on its presence? Could imagine some deeply cursed situations where someone checks in a Brewfile and it breaks windows builds.